apps/frontend/src/components/global/upload_tabs/EditEvaluationModal.vue

```diff
@@ -136,29 +137,19 @@ export default class EditEvaluationModal extends Vue {
     this.getGroupsForEvaluation();
   }
 
-  convertGroupsToIVuetifyItems(groups: IGroup[]): IVuetifyItems[] {
-    return groups.map((group) => {
-      return {
-        text: group.name,
-        value: group.id
-      }
-    })
-  }
-
   get myGroups(): IVuetifyItems[] {
     return this.convertGroupsToIVuetifyItems(GroupsModule.myGroups);
   }
 
   async getGroupsForEvaluation(): Promise<void> {
-    return axios.get<IGroup[]>(`/evaluations/${this.active.id}/groups`).then(({data}) => {
-      this.originalGroups = this.groups = this.convertGroupsToIVuetifyItems(data);
-    });
+    this.originalGroups = this.groups = this.convertGroupsToIVuetifyItems(this.active.groups);
   }
 
   async update(): Promise<void> {
-    Promise.all([EvaluationModule.updateEvaluation(this.activeEvaluation), this.updateGroups()]).then(() => {
+    await Promise.all([EvaluationModule.updateEvaluation(this.activeEvaluation), this.updateGroups()]).then(() => {
```

Comment:
Awaiting here is going to delay the visibility change. Can you just put the emit in the `then` block?